crypto/err/openssl.txt

```diff
@@ -721,7 +721,7 @@ EC_F_NISTP521_PRE_COMP_NEW:237:nistp521_pre_comp_new
 EC_F_O2I_ECPUBLICKEY:152:o2i_ECPublicKey
 EC_F_OLD_EC_PRIV_DECODE:222:old_ec_priv_decode
 EC_F_OSSL_ECDH_COMPUTE_KEY:247:ossl_ecdh_compute_key
-EC_F_OSSL_ECDSA_SIGN_SETUP:300:ossl_ecdsa_sign_setup
+EC_F_OSSL_ECDSA_SIGN_SETUP:303:ossl_ecdsa_sign_setup
```

Comment:
another glitch?

Reply:
As mentioned in that commit, this was a quick fix for inconsistency in openssl.txt introduced by 9bf682f62bd8 on the master.
I've just checked: the number clash there is gone, so I've removed this workaround.